src/pages/services/data/data.tsx

```diff
@@ -1,33 +1,39 @@
-import { ANON_IDENTITY } from "@liftedinit/many-js";
-import { Box, Button, Flex, Heading, useDisclosure } from "@liftedinit/ui";
-import { NeighborhoodContext } from "api/neighborhoods";
+import {ANON_IDENTITY} from "@liftedinit/many-js";
+import {Box, Button, Flex, Heading, useDisclosure} from "@liftedinit/ui";
+import {NeighborhoodContext} from "api/neighborhoods";
 import {
   combineData,
   useGetValues,
   useListKeys,
   useQueryValues,
 } from "api/services";
-import { useAccountsStore } from "features/accounts";
-import { useContext, useState } from "react";
-import { Breadcrumbs } from "../breadcrumbs";
-import { DataTable } from "./data-table";
-import { PutValueModal } from "./put-value-modal";
+import {useAccountsStore} from "features/accounts";
+import {useContext, useState} from "react";
+import {Breadcrumbs} from "../breadcrumbs";
+import {DataTable} from "./data-table";
+import {PutValueModal} from "./put-value-modal";
 
 export function Data() {
   const account = useAccountsStore((s) => s.byId.get(s.activeId));
   const neighborhood = useContext(NeighborhoodContext);
-  const { isOpen, onOpen, onClose } = useDisclosure();
-  const [keyvalue, setKeyvalue] = useState({ key: "", value: "" });
+  const {isOpen, onOpen, onClose} = useDisclosure();
+  const [keyvalue, setKeyvalue] = useState({key: "", value: ""});
 
-  const { data: keys } = useListKeys(neighborhood, account?.address.toString());
-  const values = useGetValues(neighborhood, keys?.keys);
-  const queries = useQueryValues(neighborhood, keys?.keys);
+  const list = useListKeys(neighborhood, account?.address.toString(),
+    [
+      {filter: [[0, account?.address.toString()], [2, false]]},               // Return all keys that are owned by address and NOT disabled
+      {filter: [[0, "maiyg"], [1, account?.address.toString()], [2, false]]}  // Return all keys that are immutable and previously owned by address and NOT disabled
```

Comment:
Ah, now I see what `params` is for. I guess I'd probably move this logic out of the view layer and into the queries. Something like:

```js
const list = useListKeys(neighborhood, account?.address.toString()));
```

And then:
```js
export function useListKeys (...snip...) {
  return [
    useQuery(...snip...), // currently owned
    useQuery(...snip...), // immutable, previously owned
  ];
}
```

Reply:
Sure, can do.